package.json

```diff
@@ -5,7 +5,7 @@
     "preinstall": "which link-module-alias && link-module-alias clean || echo",
     "start": "node build/server",
     "tsc": "tsc",
-    "dev": "cross-env NODE_ENV=development tsc-watch --onSuccess \"node ./build/server.js\"",
+    "dev": "cross-env NODE_ENV=development tsc-watch --onSuccess \"node --openssl-legacy-provider ./build/server.js\"",
```

Comment:
I think adding this flag is not the best solution, try rolling back from 20 to 16 or smth

Reply:
Reverted back to `16.17.0` and made engine limits between 12 and 17